versions/1/elasticsearch/spec.yml

```
@@ -9,7 +9,7 @@ spec:
         - description: An ML model file
           type: file
           contentMediaType: "application/json"
-          pattern: '^{PACKAGE_NAME}-.+\.json$'
+          pattern: '^{PACKAGE_NAME}_.+\.json$'
```

Or if you don't want to make this a breaking change, allow both `-` and `_`.
```suggestion
          pattern: '^{PACKAGE_NAME}[-_].+\.json$'
```

---

See comment above about how nothing is using this pattern yet so it's not a breaking change.